ios/Classes/BetterPlayerPlugin.m

```diff
@@ -193,18 +193,40 @@ - (void) setupRemoteCommands:(BetterPlayer*)player isExtraVideo:(BOOL)isExtraVid
 }
 
 - (void) setupRemoteCommandNotification:(BetterPlayer*)player, NSString* title, NSString* author , NSString* imageUrl, BOOL isLiveStream {
-    float positionInSeconds = player.position /1000;
-    float durationInSeconds = player.duration/ 1000;
-
+    // This function is always called double times at the end of video due to the default behavior of AVPlayer
+    // This check is used to prevent the latest call.
+    if (player.position >= player.duration - 500) {
+        return;
+    }
+    float positionInSeconds = player.position / 1000;
+    float durationInSeconds = player.duration / 1000;
+    BOOL isPlayingTheLastSecond = player.position >= player.duration - 1000;
 
     NSMutableDictionary * nowPlayingInfoDict = [@{MPMediaItemPropertyArtist: author,
                                                   MPMediaItemPropertyTitle: title,
                                                   MPNowPlayingInfoPropertyElapsedPlaybackTime: [ NSNumber numberWithFloat : positionInSeconds],
                                                   MPMediaItemPropertyPlaybackDuration: [NSNumber numberWithFloat:durationInSeconds],
-                                                  MPNowPlayingInfoPropertyPlaybackRate: @1,
+                                                  // Because the progress bar can auto jump to the last seek/pause/play position after reaching the end of video
```

Comment:
Sorry.
Do you know any "document" for this process?
> Because the progress bar can auto jump to the last seek/pause/play position after reaching the end of video

Reply:
@noboru-i 
Sorry, I still haven't found that document yet, but this behavior can be reproduce with [iOS sample code](https://developer.apple.com/documentation/mediaplayer/becoming_a_now_playable_app)

Reply:
Thanks. I got it.